frame/session/src/lib.rs

```diff
@@ -158,21 +158,35 @@ impl<
 }
 
 impl<
-	BlockNumber: Rem<Output=BlockNumber> + Sub<Output=BlockNumber> + Zero + PartialOrd + Saturating + Clone,
-	Period: Get<BlockNumber>,
-	Offset: Get<BlockNumber>,
-> EstimateNextSessionRotation<BlockNumber> for PeriodicSessions<Period, Offset> {
+		BlockNumber: Rem<Output = BlockNumber>
+			+ Sub<Output = BlockNumber>
+			+ Zero
+			+ PartialOrd
+			+ Saturating
+			+ Clone,
+		Period: Get<BlockNumber>,
+		Offset: Get<BlockNumber>,
```

Comment:
sidenote: I think the best syntax is to have `impl<BlockNumber, Period, Offset> ... where BlockNumber: ..., Period: ...`